### PR TITLE
fix: api/live -> api/cW9

### PR DESCRIPTION
**Pull request by justchokingaround**
# Pull Request Template

## Type of change

- [x] Bug fix
- [ ] Feature
- [ ] Documentation update

## Description

*rumbling, rumbling oh the rumbling*

## Checklist

- [x] any anime playing
- [x] bumped version
- [x] next, prev and replay work
- [x] quality works
- [x] downloads work
- [x] quality works with downloads
- [x] select episode -a and rapid resume work
- [x] syncplay -s works
- [x] autoplay, aka range selection, works

## Additional Testcases

- The safe bet: One Piece
- Episode 0: Saenai Heroine no Sodatekata ♭
- Unicode: Saenai Heroine no Sodatekata ♭
- Not uploaded: one piece dub episode 590
- Unreleased: soredemo ayumu wa yosetekuru
- Short id (for decryption): Log Horizon episode 1-2

### CODE REVIEW

**Review by CoolnsX — CHANGES_REQUESTED** [review]
Replace ```base64 -d``` with ```base64 --decode``` to enable cross compatibility with mac users..

**Comment by Derisis13** [comment]
Is the war on `'`s a compatibility feature?

**Comment by justchokingaround** [comment]
> Is the war on `'`s a compatibility feature?

just a shellcheck warning i dont like to see

**Comment by Derisis13** [comment]
This one is starting to get big. Either make it a .N release (so it'll be packaged) or wait for some time after merging it (testing) and just up the version number

**Review by CoolnsX — APPROVED** [review]

**Review by CoolnsX — COMMENTED** [review]
Replace
```sed -nE 's_location: (.*)_\1_p```
with
```sed -nE 's_[L|l]ocation: (.*)_\1_p```
In line
https://github.com/pystardust/ani-cli/blob/df04506e25e43c71d2e109d45b19af5f589f5b41/ani-cli#L209

File: ani-cli [hunk 1]
```diff
@@ -216,11 +216,11 @@ generate_link() {
...
 			iv=$(printf "%s" "$resp" | sed -n '3p' | tr -d "\n" | od -A n -t x1 | tr -d " |\n")
 			second_key=$(printf "%s" "$resp" | sed -n '4p' | tr -d "\n" | od -A n -t x1 | tr -d " |\n")
-			token=$(printf "%s" "$resp" | head -1 | base64 -d | openssl enc -d -aes256 -K "$secret_key" -iv "$iv" | sed -nE 's/.*&(token.*)/\1/p')
+			token=$(printf "%s" "$resp" | head -1 | base64 --decode | openssl enc -d -aes256 -K "$secret_key" -iv "$iv" | sed -nE 's/.*&(token.*)/\1/p')
 			ajax=$(printf '%s' "$id" | openssl enc -e -aes256 -K "$secret_key" -iv "$iv" -a)
 			data=$(curl -A "$agent" -sL -H "X-Requested-With:XMLHttpRequest" "${gogohd_url}encrypt-ajax.php?id=${ajax}&alias=${id}&${token}" | sed -e 's/{"data":"//' -e 's/"}/\n/' -e 's/\\//g')
-			result_links="$(printf '%s' "$data" | base64 -d 2>/dev/null | openssl enc -d -aes256 -K "$second_key" -iv "$iv" 2>/dev/null |
-				sed -e 's/\].*/\]/' -e 's/\\//g' | grep -Eo 'https:\/\/[-a-zA-Z0-9@:%._\+~#=][a-zA-Z0-9][-a-zA-Z0-9@:%_\+.~#?&\/\/=]*')"
+			result_links="$(printf '%s' "$data" | base64 --decode 2>/dev/null | openssl enc -d -aes256 -K "$second_key" -iv "$iv" 2>/dev/null |
+				tr -d "\\" | sed -nE "s_.*file\":\"([^\"]*)\".*source.*_\1_p")"
```
Comment (Derisis13) [inline ani-cli:223]: This one needs some work, I just got a warning:
```
Fetching Gogoanime Direct link..tr: warning: an unescaped backslash at end of string is not portable
```
A possible fix is written here: https://cygwin.com/pipermail/cygwin/2018-December/239487.html

**Review by Derisis13 — CHANGES_REQUESTED** [review]
That tr warning might cause some confusion (and delay) and might even be an issue on other platforms

File: ani-cli (hunk 1 above)
Comment (CoolnsX) [reply ani-cli:223]: Yus.. it should be in single qutoes..
like this ```tr -d '\\'```

Comment (justchokingaround) [reply ani-cli]: fixed in latest commit, sorry for the delay

Comment (justchokingaround) [reply ani-cli]: welp, shellcheck didn't like that

**Review by Derisis13 — APPROVED** [review]
Works flawlessly. If no issues arise in a week we should consider making this state 3.5 without changes.

**Pull request merged**